src/configurations/destinations/bingads_audience/ui-config.json

```diff
@@ -1,5 +1,16 @@
 {
   "uiConfig": [
+    {
+      "title": "Account authentication",
+      "fields": [
+        {
+          "type": "accountManagementInput",
+          "label": "Event delivery account",
+          "value": "rudderAccountId",
```

Comment:
Shouldn't  this be `configKey` instead of `value`?

Reply:
In old form builder it is value and in new it is configKey